tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrationUtil.java

```diff
@@ -481,7 +481,15 @@ public static void problemToMarker( Problem problem, IMarker marker ) throws Cor
         marker.setAttribute( "migrationProblem.autoCorrectContext", problem.getAutoCorrectContext() );
 
         marker.setAttribute( IMarker.LOCATION, problem.file.getName() );
-        marker.setAttribute( IMarker.SEVERITY, IMarker.SEVERITY_ERROR );
+
+        if( problem.summary.contains( "@deprecated As of 7.0.0" ) )
```

Comment:
Lets not hardcode this to look for a specific string, that leads to brittle code in the future.  Can you propose a better design alternative?  Thanks.

Reply:
thanks, got it